core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -987,6 +987,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
 
   def batchUpdateEntities(workspaceName: WorkspaceName, entityUpdates: Seq[EntityUpdateDefinition], upsert: Boolean = false): Future[PerRequestMessage] = {
+    logger.info(s"Inside batchUpdateEntities workspaceName $workspaceName, with entities $entityUpdates with upsert $upsert")
```

Comment:
residual logs from debugging in this file?

Reply:
yes :)